docs/use-cases/hydra-for-payments/index.md

```diff
@@ -0,0 +1,43 @@
+# Hydra for Payments
+
+> Using Hydra for Peer-To-Peer Payments
```

Comment:
These subtitles are a bit pointless as they just re-state the other headline(s).

Should: remove it